osu.Game/Beatmaps/ControlPoints/DifficultyControlPoint.cs

```diff
@@ -7,17 +7,20 @@
 
 namespace osu.Game.Beatmaps.ControlPoints
 {
+    /// <remarks>
+    /// Note that going forward, this control point type should always be assigned directly to HitObjects.
+    /// </remarks>
     public class DifficultyControlPoint : ControlPoint
     {
         public static readonly DifficultyControlPoint DEFAULT = new DifficultyControlPoint
         {
-            SpeedMultiplierBindable = { Disabled = true },
+            SliderVelocityBindable = { Disabled = true },
         };
 
         /// <summary>
-        /// The speed multiplier at this control point.
+        /// The slider velocity at this control point.
         /// </summary>
-        public readonly BindableDouble SpeedMultiplierBindable = new BindableDouble(1)
+        public readonly BindableDouble SliderVelocityBindable = new BindableDouble(1)
```

Comment:
It seems weird to call this "SliderVelocity" when it's something fundamental to hitobjects and should probably be named in a general way as such.

What's the path forward here? Is it to eventually remove `DifficultyControlPoint` and give `Slider` a "Velocity" property, or is it to continue using `DifficultyControlPoint` for hitobjects? If the latter, then I propose this to be renamed to just "Velocity".

Reply:
yeah, next step is to nuke this completely. can rename i guess, though much of a muchness. i feel SliderVelocity gave a better idea of direction?

Reply:
I think it's fine for now in that case.